charts/newrelic-infrastructure/templates/controlplane/daemonset.yaml

```diff
@@ -18,63 +19,63 @@ spec:
   {{- end }}
   selector:
     matchLabels:
-      {{- include "common.labels.selectorLabels" . | nindent 6 }}
+      {{- include "newrelic.common.labels.selectorLabels" . | nindent 6 }}
       app.kubernetes.io/component: controlplane
   template:
     metadata:
       annotations:
         checksum/nri-kubernetes: {{ include (print $.Template.BasePath "/controlplane/scraper-configmap.yaml") . | sha256sum }}
         checksum/agent-config: {{ include (print $.Template.BasePath "/controlplane/agent-configmap.yaml") . | sha256sum }}
-        {{- if include "common.license.secret" . }}{{- /* If the is secret to template */}}
+        {{- if include "newrelic.common.license.secret" . }}{{- /* If the is secret to template */}}
```

Comment:
```suggestion
        {{- if include "newrelic.common.license.secret" . }}
        {{- /* Add checksum only if we are not using an external secret */}}
```

Reply:
I'm still not happy about checksumming a secret though 🤔

Reply:
The secret has also the labels: https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_license_secret.yaml.tpl#L17

So it is salted. So I see these issues:
 * If we use a random string that uses a seed, we still have the same issue that it can be brute-forced.
 * If we put a pure random string, any change on the helm chart will redeploy everything.
 
 I see no solution that could make a horrible UX.

Reply:
Me neither, the only solution I see is just not force a redeploy if LK changes (which on the other hand I don't see as a very bad thing).